use-cases/grpc_python/Makefile

```diff
@@ -26,3 +29,25 @@ run:
 
 down:
 	PIPELINE_COUNT=$(PIPELINE_COUNT) MODEL_NAME=$(MODEL_NAME) RETAIL_USE_CASE_ROOT="$(RETAIL_USE_CASE_ROOT)" RESULTS_DIR="$(RESULTS_DIR)" docker compose -f $(DOCKER_COMPOSE) down
```

Comment:
get an error on this **make down** command:
```console
jimwang@jimwang-NUC11TNHv7:~/go/src/github.com/brian/retail-use-cases/use-cases/grpc_python$ make down
PIPELINE_COUNT=1 MODEL_NAME=instance-segmentation-security-1040 RETAIL_USE_CASE_ROOT="../.." RESULTS_DIR="../../results" docker compose -f docker-compose_grpc_python.yml down
WARN[0000] /home/jimwang/go/src/github.com/brian/retail-use-cases/use-cases/grpc_python/docker-compose_grpc_python.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
service "ovms-client-grpc-python" depends on undefined service "OvmsServer": invalid compose project
make: *** [Makefile:31: down] Error 15

```